site/_docs/configuration.md

```diff
@@ -351,27 +351,27 @@ before your site is served.
 
 ## Specifying a Jekyll environment at build time
 
-In the build (or serve) arguments, you can specify a Jekyll environment and value. The build will then apply this value in any conditional statements in your content. 
+In the build (or serve) arguments, you can specify a Jekyll environment and value. The build will then apply this value in any conditional statements in your content.
 
 For example, suppose you set this conditional statement in your code:
 
-```liquid
- {% raw %}
- {% if jekyll.environment == "production" %}
-   {% include disqus.html %} 
- {% endif %}
- {% endraw %}
-```
+{% highlight ruby %}
```

Comment:
@parkr I wasn't sure. I looked at the templates page and it [used `ruby` and `text`](https://github.com/jekyll/jekyll/blame/master/site/_docs/templates.md#L270) for Liquid highlighting. Should I update?